erpnext/education/doctype/article/article.json

```diff
@@ -33,9 +36,17 @@
    "fieldname": "publish_date",
    "fieldtype": "Date",
    "label": "Publish Date"
+  },
+  {
+   "fieldname": "course",
```

Comment:
Thanks for your notes. the reason behind this is if we would allow same title for content, then we need to distinguish between them. Adding course here will allow this and as it is not mandatory it will not affect the flow you mentioned above.
![3](https://user-images.githubusercontent.com/59777434/109410067-99fea080-79a8-11eb-823d-3ffdb10b7892.png)

as shown in the screenshot. we have three courses with the same content title.